INSTALL.md

```diff
@@ -3,7 +3,7 @@
 These instructions are designed for setting up The Rails Port for development and testing.
 If you want to deploy the software for your own project, then see the notes at the end.
 
-You can install the software directly on your machine, which is the traditional and probably best-supported approach. However, there is an alternative which may be easier: Vagrant. This installs the software into a virtual machine, which makes it easier to get a consistent development environment and may avoid installation difficulties. For Vagrant instructions, see [VAGRANT.md](VAGRANT.md).
+You can install the software directly on your machine following the instructions below. Alternatively there are guides to use either [Vagrant](VAGRANT.md) or [Docker](DOCKER.md).
```

Comment:
I would still call running software directly on your machine the traditional and probably best-supported approach.